README.md

```diff
@@ -38,7 +38,7 @@ exports.handler = lumigo.trace(myHandler)
 
 ## Configuration
 * You can turn on the debug logs by setting the environment variable `LUMIGO_DEBUG=TRUE`
-* You can prevent lumigo from sending keys that answer specific regexes by defining `LUMIGO_SECRET_MASKING_REGEX=["regex1", "regex2"]`. By default, we use the default regexes `[".*pass.*", ".*key.*", ".*secret.*", ".*credential.*", ".*passphrase.*"]`. All the regexes are case-insensitive.
+* You can prevent lumigo from sending keys that answer specific regexes by defining `LUMIGO_SECRET_MASKING_REGEX=["regex1", "regex2"]`. By default, we use the default regexes `[".pass.", ".key.", ".secret.", ".credential.", ".passphrase.", "SessionToken", "x-amz-security-token", "Signature", "Credential", "Authorization"]`. All the regexes are case-insensitive.
```

Comment:
same as in the other PR.
also here, note that we wrote that this regex is case insensitive, so maybe change everything lowercase :)